src/main/java/org/protege/editor/owl/client/LocalHttpClient.java

```diff
@@ -873,15 +874,31 @@ public boolean codeIsLessThan(String lower, String upper) {
 	
 	}
 	
-	public boolean isWorkFlowManager() { 
-    	
-    		try {
-    			Role wfm = getRole(new RoleIdImpl("mp-project-manager"));
-    			return getActiveRoles().contains(wfm);
-    		} catch (ClientRequestException e) {
-    			e.printStackTrace();
-    			return false;
-    		}
-    	
+	public boolean isWorkFlowManager() {
+		try {
+			Role wfm = getRole(new RoleIdImpl("mp-project-manager"));
+			return getActiveRoles().contains(wfm);
+		} catch (ClientRequestException e) {
+			e.printStackTrace();
+			return false;
+		}
     }
+
+    public ServerStatus getServerStatus() throws ClientRequestException {
+		Response response = null;
+		try {
+			response = get(SERVER_STATUS);
```

Comment:
Its in
src/main/java/org/protege/editor/owl/server/http/ServerEndpoints.java
in the protege-server pr